pandas/tseries/frequencies.py

```diff
@@ -149,7 +138,7 @@ def get_freq_group(cls, resostr):
         """
         return get_freq_group(cls.get_freq(resostr))
 
-    @classmethod
+    @classmethod  # noqa:F811  # Disable flake8 false-positive
```

Comment:
pls don't use flake warnings like this

Reply:
See discussion from 8 days ago:

> For reasons unknown, flake8 was producing incorrect warnings about the function get_freq imported above being redefined. Here and on line 141.

The easiest way to make this these unnecessary is probably #18034.

Reply:
ok